projects/1.20/assets/blue-skies/blue_skies/lang/zh_cn.json

```
@@ -0,0 +1,1354 @@
+	"advancements.blue_skies.get_pyrope": "迅捷升级",
+	"advancements.blue_skies.get_pyrope.desc": "挖掘榴石矿石以获得一块宝石。",
+	"advancements.blue_skies.get_aquite": "巨大提升",
+	"advancements.blue_skies.get_aquite.desc": "将粗水蓝石烧炼为水蓝石。",
+	"advancements.blue_skies.get_diopside": "坚不可摧",
+	"advancements.blue_skies.get_diopside.desc": "挖掘透辉石矿石以获得一块宝石。",
+	"advancements.blue_skies.get_charoite": "紫色力量！",
+	"advancements.blue_skies.get_charoite.desc": "将粗绛紫晶烧炼为绛紫晶。",
+	"advancements.blue_skies.everbright.get_falsite": "神秘金属",
+	"advancements.blue_skies.everbright.get_falsite.desc": "将粗谬铁烧炼成锭。",
+	"advancements.blue_skies.everbright.get_ventium": "稍带点改装红色",
+	"advancements.blue_skies.everbright.get_ventium.desc": "将粗威红铁烧炼成锭。",
+	"advancements.blue_skies.everdawn.get_horizonite": "内心温暖",
+	"advancements.blue_skies.everdawn.get_horizonite.desc": "将粗炙铁烧炼成锭。",
+
+	"entity.minecraft.villager.blue_skies.stargazer": "观星者",
+	"entity.minecraft.villager.blue_skies.nightwatcher": "守夜人",
+	"entity.minecraft.villager.blue_skies.wrangler": "牧人",
+	"entity.minecraft.villager.blue_skies.shoveler": "铲雪人",
+	"entity.minecraft.villager.blue_skies.summoner": "召唤师",
+	"entity.minecraft.villager.blue_skies.alchemist": "炼金师",
+
+	"death.attack.blue_skies:bite": "%1$s被%2$s咬死了",
+	"death.attack.blue_skies:bite.player": "%1$s在试图逃离%2$s时被咬死了",
+	"death.attack.blue_skies:rock_stab": "%1$s被尖刺扎死了",
+	"death.attack.blue_skies:rock_stab.mob": "%1$s被%2$s用尖刺扎死了",
+	"death.attack.blue_skies:rock_stab.player": "%1$s在与%2$s战斗时被尖刺扎死了",
+	"death.attack.blue_skies:venom_spit": "%1$s中了致命的剧毒",
+	"death.attack.blue_skies:venom_spit.mob": "%1$s因%2$s中了致命的剧毒",
+	"death.attack.blue_skies:venom_spit.player": "%1$s在与%2$s战斗时中了致命的剧毒",
+	"death.attack.blue_skies:summoned_lightning": "%1$s被%2$s击中而死",
+	"death.attack.blue_skies:fluctuant_sphere": "%1$s被%2$s震击而死",
+	"death.attack.blue_skies:ember": "%1$s无法承受%2$s的热量",
+	"death.attack.blue_skies:summon": "%1$s被%2$s的%3$s杀死了",
+  	"death.attack.blue_skies:summon.item": "%1$s被%2$s的%3$s用%4$s杀死了",
+	"death.attack.blue_skies:slam": "%1$s被%2$s抛向了天空",
+	"death.attack.blue_skies:slam.player": "%1$s在与%2$s战斗时被抛向了天空",
+	"death.attack.blue_skies:root": "%1$s被%2$s用根缠绕而亡",
+	"death.attack.blue_skies:root.player": "%1$s在与%2$s战斗时走入了根系之中",
+
+	"gui.blue_skies.item.falsite_uses": "谬铁耐久度：%s / %s",
+	"gui.blue_skies.item.spear.when_thrown": "投掷出时：",
+	"gui.blue_skies.item.spear.thrown_damage": " %s 攻击伤害",
+	"gui.blue_skies.item.valid_materials": "材料",
+	
+	"gui.blue_skies.item.ability": "能力",
+	"gui.blue_skies.item.ability.missing": "尚未实装",
+	"gui.blue_skies.item.ability.summon_boss": "更高难度下可在钥匙孔处重新召唤Boss。",
+
+	"gui.blue_skies.item.ability.runic.0": "将佩戴的所有饰品的稀有度增加1。",
+
+	"gui.blue_skies.item.ability.ethereal.0": "移动速度增加10%%。",
+	"gui.blue_skies.item.ability.ethereal.1": "移动速度增加15%%。",
+	"gui.blue_skies.item.ability.ethereal.2": "移动速度增加20%%。",
+	"gui.blue_skies.item.ability.ethereal.3": "移动速度增加25%%。",
+
+	"gui.blue_skies.item.ability.dusk.0": "潜行时隐形，会产生黑烟。",

```

```suggestion
	"gui.blue_skies.item.ability.dusk.0": "潜行时隐形，并产生黑烟。",
```
自己玩的时候总感觉`会`有割裂感
<s>主要是更高等级的很难刷到手</s>